components/dropdown/Dropdown.js

```diff
@@ -98,6 +112,38 @@ const factory = (Input) => {
         : undefined;
     };
 
+    getNextSelectableItemIndex = (focusedItemIndex) => {
+      const { source } = this.props;
+      const lastItemIndex = source.length - 1;
+
+      // Set the next index
+      let nextIndex = focusedItemIndex !== lastItemIndex ? focusedItemIndex + 1 : 0;
+
+      // If the next item is disabled then keep going until we either find a non-disabled item or we get back to the
+      // original focused item
+      while (source[nextIndex].disabled && nextIndex !== focusedItemIndex) {
+        nextIndex = nextIndex !== lastItemIndex ? nextIndex + 1 : 0;
+      }
+
+      return nextIndex;
+    };
```

Comment:
Can be a little shorter:
```js
getNextSelectableItemIndex = (focusedItemIndex) => {
    const { source } = this.props;
    const lastItemIndex = source.length - 1;
    let nextIndex = focusedItemIndex;

    // If the next item is disabled then keep going until we either find a non-disabled item or we get back to the
    // original focused item
    do {
      nextIndex = nextIndex !== lastItemIndex ? nextIndex + 1 : 0;
    } while (source[nextIndex].disabled && nextIndex !== focusedItemIndex);

    return source[nextIndex];
}
```

Reply:
@fantactuka, thanks. Not sure if this is getting merged anyway, so I'll wait on making any changes.